otexplorer/src/print.rs

```diff
@@ -27,7 +27,7 @@ impl<'a> std::io::Write for PrettyPrinter<'a> {
     fn write(&mut self, buf: &[u8]) -> std::io::Result<usize> {
         let len = self.writer.write(buf)?;
         let wrote_buf = &buf[..len];
-        let wrote_str = unsafe { std::str::from_utf8_unchecked(wrote_buf) };
```

Comment:
this won't compile since the type has changed from `&str` to `Result<&str, _>`. I agree this should be changed, but let's just do `String::from_utf8_lossy`?

Reply:
Oh my bad, thanks.